test/coreneuron/test_spikes.py

```diff
@@ -24,9 +24,10 @@
     from neuron import h, gui
 # without mpi4py we need to call nrnmpi_init explicitly
 elif nrnmpi_init_option:
-    from neuron import h, gui
+    from neuron import h
 
     h.nrnmpi_init()
+    from neuron import gui  # if mpi is active, don't ask for it til gui turned off
```

Comment:
Why import the GUI if the GUI is turned off?

Reply:
It would suffice in this case. I changed the comment since ```h.nrnmpi_init()``` now turns off gui for all ranks > 0